tutorials/docker/Dockerfile

```diff
@@ -1,27 +1,59 @@
-FROM rticom/connext-sdk:7.3.0-EAR as build-stage
+ARG RTI_LICENSE_AGREEMENT_ACCEPTED=false
```

Comment:
Instead of adding all this to this Dockerfile, my idea was to define a new Dockerfile, (e.g. `connext.Dockerfile`) that creates the `connext-sdk` image: 
```
docker build -t connext-sdk ./connext.Dockerfile`
```

(The git clone could be part of this image as well)

And then keep this Dockerfile very similar (starting with `FROM connext-sdk as build-stage`).

Reply:
I see, I'll decouple the connext-sdk building part into a different dockerfile and then try to link both of them back together